packages/kubernetes-api-app/package.json

```diff
@@ -11,12 +11,12 @@
   },
   "dependencies": {
     "@hawtio/online-kubernetes-api": "workspace:*",
-    "@hawtio/react": "^0.6.1",
-    "@patternfly/react-core": "^4.276.8",
+    "@hawtio/react": "^0.9.1",
```

Comment:
Please upgrade to 0.9.2.

Reply:
See https://github.com/hawtio/hawtio-online/pull/199